### DISCUSSION TITLE
RetroArch

### DISCUSSION THREAD

**Discussion by mudkipdev** [catppuccin/catppuccin#2116] [Port Requests]
### Will you make the port?

- [ ] Yes, I'm willing to make the port

### How can the suggested port be themed?

https://docs.libretro.com/guides/themes/

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by TheGamerCanine** [comment]
I'd love to see a Catppuccin theme for RetroArch! The Dracula theme suffices for now but it would be really cool to see a complete match for my system.
